tests/queries/0_stateless/00718_format_datetime_1.sql

```diff
@@ -3,3 +3,4 @@ select formatDateTime(toDateTime64('1962-12-08 18:11:29.123', 3, 'UTC'), '%F %T.
 select formatDateTime(toDateTime64('1969-12-31 23:59:59.999', 3, 'UTC'), '%F %T.%f');
 select formatDateTime(toDateTime64('1970-01-01 00:00:00.000', 3, 'UTC'), '%F %T.%f');
 select formatDateTime(toDateTime64('1970-01-01 00:00:00.001', 3, 'UTC'), '%F %T.%f');
+select formatDateTime(toDateTime64('2024-12-11 12:34:56.000', 3, 'UTC'), '%W%z');
```

Comment:
In general new test cases should be created in a new file. Let's do it and add a couple more tests with different time zones (both negative and positive), so we'll have this check for all the cases in the future